app/src/main/java/com/x8bit/bitwarden/data/auth/datasource/network/service/IdentityServiceImpl.kt

```diff
@@ -43,10 +43,6 @@ class IdentityServiceImpl(
                         codes = listOf(400, 429),
                         json = json,
                     )
-                    ?: bitwardenError.parseErrorBodyOrNull<RegisterResponseJson.Error>(
```

Comment:
So this is what will never be hit because `Invalid` will always fulfill the contract for `Error`